packages/components/pages/f-loyalty/src/tenants/en-AU.js

```diff
@@ -33,6 +33,14 @@ const messages = {
         loginButtonText: 'Log in'
     },
     howItWorks: {
+        imagePath: 'https://just-eat-prod-eu-res.cloudinary.com/image/upload/v1630068495/Experiments/Homeweb-Coreweb/',
+        bagValues: [
```

Comment:
Not sure I understand the logic of the bagValues, can you please explain them?

Reply:
each tenant has their own amounts they want shown in the bags diagram in how it works page, these numbers are used to find the correct bag image to show.

Reply:
Yeah had already made the connection with the images, was more about what the values in the array represent.
```
 bagValues: [
            40,
            50,
            45,
            40,
            55
        ],
```

So what relationship between the $40 image and the order value for example?

Reply:
$40 is $40 order value. Each market wants different order values as examples.

Reply:
Ah excellent, thanks